package/thingino-portal/files/index.cgi

```diff
@@ -53,7 +53,7 @@ elif [ "GET" = "$REQUEST_METHOD" ] || [ "edit" = "$POST_mode" ]; then %>
 <div class="mb-3">
 <label class="form-label">Wireless Network Password</label>
 <input class="form-control form-control-lg bg-light text-dark" type="text" name="wlanpass" id="wlanpass" value="<%= $wlanpass %>" required autocapitalize="none">
```

Comment:
maybe we could do something like
`minlength="8" pattern=".{8,64}"` to enforce the requirement in the input box?

Reply:
seems i added it in my device but not the pr lol.
`pattern=".{8,64}"` works better than `maxlength="64"`

Added and updated